partd/pandas.py

```diff
@@ -14,6 +15,10 @@
 
 dumps = partial(pickle.dumps, protocol=pickle.HIGHEST_PROTOCOL)
 
+_NO_EXTENSION = 'numpy_type'
+_EXTENSION_CATEGORICAL = 'categorical_type'
+_EXTENSION_DATETIME_WITH_TZ = 'datetime64_tz_type'
```

Comment:
Maybe just replace `_FOO` with `'foo'`?  The indirection here may not be necessary.